src/features/utils/lazy-loader.js

```diff
@@ -33,26 +33,5 @@ export function lazyLoader (featureName, featurePart) {
       default:
         throw new Error(`Attempted to load unsupported agent feature: ${featureName} ${featurePart}`)
     }
```

Comment:
Add a comment in this file explaining why we should only import "Aggregate" files